Diffing_Engine/Compute/_IDiffing.cs

```diff
@@ -43,7 +43,7 @@ namespace BH.Engine.Diffing
     public static partial class Compute
     {
         [Description("Dispatches to the most appropriate Diffing method, depending on the provided inputs.")]
-        public static Diff Diffing(IEnumerable<object> pastObjs, IEnumerable<object> followingObjs, DiffConfig diffConfig = null, string customDataIdKey = null)
+        public static Diff Diffing(IEnumerable<object> pastObjs, IEnumerable<object> followingObjs, DiffingType diffingType = DiffingType.Automatic, DiffingConfig diffConfig = null)
```

Comment:
File name probably shouldn't be starting with an `_` - is there a reason why it does @alelom? I think removing `_` will allow compliance to be happy, as it takes into account `I` files I think.

Reply:
Oh, I was using it simply to have the file on top in my folder in VS. Sorry I didn't notice it was a problem. Removing it.